.github/actions/pr-insert-description/action.yml

```
@@ -0,0 +1,51 @@
+runs:
+  using: 'composite'
+  steps:
+    - name: 'Check for label and update PR description'
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        # PR番号を取得
+        PR_NUMBER=$(jq -r '.pull_request.number' "$GITHUB_EVENT_PATH")
+        # PRのラベルを取得
+        LABELS=$(jq -r '.pull_request.labels[].name' "$GITHUB_EVENT_PATH")
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)